test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/TestPhaseIRMatching.java

```diff
@@ -555,24 +555,24 @@ public static List<Failure> sort(Set<Failure> failures) {
 class LoadStore {
     int i;
     float f;
-    interface I1{}
+    interface I1 {}
     static class Base implements I1 {
         int i;
     }
-    interface I2{}
+    interface I2 {}
     static class Derived extends Base implements I2 {
         long l;
     }
-    Base Lb = new Base();
-    Derived Ld = new Derived();
+    Base base = new Base();
+    Derived derived = new Derived();
 
-    static class SingleNest{
-        static class DoubleNest{
+    static class SingleNest {
+        static class DoubleNest {
             int i;
         }
     }
 
-    SingleNest.DoubleNest Ldn = new SingleNest.DoubleNest();
+    SingleNest.DoubleNest double_nest = new SingleNest.DoubleNest();
```

Comment:
Last nit: We should use camelCase for Java code:
```suggestion
    SingleNest.DoubleNest doubleNest = new SingleNest.DoubleNest();
```